tests/apidoc_login_test.go

```diff
@@ -81,6 +82,70 @@ func TestLogin(t *testing.T) {
 				},
 			})
 		})
+		// sytest: POST /login can log in as a user with just the local part of the id
+		t.Run("POST /login can log in as a user with just the local part of the id", func(t *testing.T) {
```

Comment:
We have actually been doing this the entire time! You should go back to the previous tests:
 - POST /login can login as user
 - POST /login returns the same device_id as that in the request

 and add `@` and `:hs1` to the `user` key. E.g:
```
"user": "device_id_test_user"
"user": "@device_id_test_user:hs1"
```

Sorry I didn't catch that earlier.

Reply:
@Kegsay  should I return make the createDummyUser return the UserID from the server response and then use that userID as was done in the perl tests?

Reply:
Yup that sounds good!